api/controllers/user.js

```diff
@@ -447,19 +447,18 @@ async function storePassword(req, res) {
       userId: userid,
       status: false,
     }).exec();
-    if (!resetPassword) {
+    const expireTime = moment.utc(resetPassword.resetPasswordExpires).isBefore(moment());
```

Comment:
I'm not sure it's safe to compare a UTC datetime with a local datetime.

Reply:
I will try to fix it :)